relay-server/src/envelope/container.rs

```diff
@@ -146,147 +242,91 @@ impl<T: ContainerItem> ItemContainer<T> {
 
     fn deserialize<'de, D: de::Deserializer<'de>>(deserializer: D) -> Result<Self, D::Error> {
         #[derive(Deserialize)]
-        struct Layout<T> {
-            #[serde(bound(deserialize = "T: FromValue"))]
-            items: AnnotatedItems<T>,
+        #[serde(bound(deserialize = "T: ContainerItem"))]
```

Comment:
Is this annotation actually necessary?

Reply:
Yes:

![image](https://github.com/user-attachments/assets/487c612d-cffe-4e83-9957-f4fa3b32e215)